CMakeLists.txt

```diff
@@ -31,8 +31,8 @@ endif()
 # Increment as appropriate after release of a new version, and set back
 # AUDACITY_BUILD_LEVEL to 0
 set( AUDACITY_VERSION 3 )
-set( AUDACITY_RELEASE 2 )
-set( AUDACITY_REVISION 0 )
+set( AUDACITY_RELEASE 1 )
+set( AUDACITY_REVISION 1 )
```

Comment:
Remove these changes from the branch please

Reply:
resolved with rebase